torchreid/models/ml_decoder.py

```diff
@@ -0,0 +1,171 @@
+from typing import Optional
```

Comment:
I'd place the original license here  as Leonid did for other files. The same is for GCN.

Reply:
I added MIT